packages/block-editor/src/store/effects.js

```diff
@@ -66,6 +67,57 @@ export function validateBlocksToTemplate( action, store ) {
 	}
 }
 
+/**
+ * Announces when a block is moved.
+ *
+ * @param {Object} state The current state.
+ * @param {string} direction The direction the block was moved in.
+ */
+export function announceBlockMoved( state, direction ) {
+	const blockCount = getSelectedBlockCount( state );
+	let message;
+
+	switch ( direction ) {
```

Comment:
It seems all these could be inlined and don't need the extra abstraction of `announceBlockMoved`

Reply:
I'd disagree. Seems to me it's way more readable this way.